Configuration/PyReleaseValidation/python/relval_standard.py

```diff
@@ -130,6 +130,7 @@
 workflows[140.55] = ['',['RunHI2015VR','HYBRIDRepackHI2015VR','HYBRIDZSHI2015','RECOHID15','HARVESTDHI15']]
 workflows[140.56] = ['',['RunHI2018','RECOHID18','HARVESTDHI18']]
 workflows[140.57] = ['',['RunHI2018Reduced','RECOHID18','HARVESTDHI18']]
+workflows[140.61] = ['',['RunHI2018AOD','REMINIAODHID18','HARVESTHI18MINIAOD']]
```

Comment:
is there an exact matching rereco workflow number for this one?
In my #29756 I proposed to reuse the existing full reco wf index as much as possible.
In the pp data processing case, the UL pp remini workflows have "11" appended.
The index can be anything, but I just thought that this way it could be easier to match or recognize the workflows.